qiskit/pulse/instructions/delay.py

```diff
@@ -67,7 +67,3 @@ def channels(self) -> Tuple[Channel]:
     def duration(self) -> Union[int, ParameterExpression]:
         """Duration of this instruction."""
         return self.operands[0]
-
-    def is_parameterized(self) -> bool:
```

Comment:
This is because base class method (`Instruction.is_parameterized`) is updated. Since all pulse instructions take the from of `Class(*operands)`, we don't need to write a dedicated logic to investigate operands for each class. We just need to write a base class logic that investigates parameters of `self._operands`.